content/en/case-studies/blackhole-image.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Case Study: The First Image of a Black Hole, ever!"
+sidebar: false
+---
+
+<img
+  src="/images/content_images/cs/blackhole.jpg"
+  alt="bh"
+  title="Black Hole M87">
+
+```
+Imaging the M87 Black Hole is like trying to see something that is by definition impossible to see!
```

Comment:
This is visible now, but it's too wide so we see a scroll box. I suggest to put a line break after _something_

Reply:
Let me check, it was all showing up properly in safari.